Update anticosti-historical-data-tutorial.Rmd

vmglynn opened the pull request:
We wrote a short introduction and linked our shinyapp tutorial. For the later, if you prefer we do it differently, please let us know!

vmglynn committed:
Update anticosti-historical-data-tutorial.Rmd

graciellehigino commented on _lessons/2022-04-07-anticosti-historical-data-tutorial/anticosti-historical-data-tutorial.Rmd:
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Anticosti Historical Data Tutorial"
+title: "After the storm: Impacts of a hemlock looper outbreak on tree species in Anticosti Island, 1973"
```
Maybe the title could be shorter and the new title you added would go in the "description" filed... What do you think? I really liked it, by the way!

vmglynn replied:
@graciellehigino - I think for the actual tutorial, it would be best to keep the longer descriptive title, but for the webpage's entry I think the short tutorial sounds great! What do you think @quantitative-ecologist ?

quantitative-ecologist replied:
I don't really mind. It's nice to have a long title to convey the story, but I think that for the tutorial in general, the short title would be good for the website as you suggested @vmglynn.

graciellehigino commented on _lessons/2022-04-07-anticosti-historical-data-tutorial/anticosti-historical-data-tutorial.Rmd:
```diff
@@ -11,11 +11,22 @@ output:
+
+[Here is a direct link to our tutorial](https://vmglynn-ldp.shinyapps.io/LDT_Anticosti_Tutorial/#section-welcome-to-the-living-data-tutorials)
+
+The present tutorial was developed by Victoria Marie Glynn who is a PhD student at McGill University in Montréal, Québec, and Maxime Fraser Franco who is a PhD student at the Université du Québec à Montréal (UQAM) in Montréal, Québec. Please feel free to contact us if you have any question.
```
I think it would be cleaner if we added affiliations in the YAML like this:
```
author:
  - name: Your Name
    affiliation: Your Institution
```

Maybe this works for email too? Like, adding a tag under `affiliation` as `url` followed by a mailto: link to your email.

vmglynn replied:
Sounds good, I will do this now @graciellehigino ! Thank you for the suggestion!

graciellehigino reviewed (changes requested):
Awesome job, team! Thank you so much for the improvements. I've made some comments, let me know if you want to edit the files even further or if I can merge the changes as they are.

graciellehigino commented on _lessons/2022-04-07-anticosti-historical-data-tutorial/anticosti-historical-data-tutorial.Rmd:
```diff
@@ -11,11 +11,22 @@ output:
+
+The tutorial was built with the free and open-source software environment [R](https://www.r-project.org/) using the [leanr package](https://rstudio.github.io/learnr/) and and deployed using [Shiny](https://shiny.rstudio.com/). You can find the project folders and files in this [GitHub repository](https://github.com/Living-Data-Tutorials/Anticosti-Historical-Data).
+
+[Here is a direct link to our tutorial](https://vmglynn-ldp.shinyapps.io/LDT_Anticosti_Tutorial/#section-welcome-to-the-living-data-tutorials)
```
```suggestion

```

vmglynn replied:
This amended section looks good to me, thanks for the suggestion @graciellehigino. What do you think @quantitative-ecologist ?

quantitative-ecologist replied:
Ok so you removed the link to the tutorial so we can embedded it instead?

vmglynn replied:
I think Gracielle was able to do this -- thank you very much @graciellehigino !

vmglynn committed:
Gracielle edits - Anticosti Rmd

Anticosti Rmd with Gracielle's suggestions incorporated: changing the title, adding our affiliations/emails in the YAML header

Commit:
Merge pull request #16 from Living-Data-Tutorials/Gracielle-edits---Anticosti-Rmd

Gracielle edits - Anticosti Rmd
